lib/billy.rb

```diff
@@ -42,11 +42,11 @@ def self.register_drivers
 
     if defined?(Capybara::Webkit::Driver)
       Capybara.register_driver :webkit_billy do |app|
-        driver = Capybara::Webkit::Driver.new(app)
-        driver.browser.ignore_ssl_errors
-        driver.browser.set_proxy(host: Billy.proxy.host,
-                                 port: Billy.proxy.port)
-        driver
+        options = {
+          :ignore_ssl_errors => false,
+          :proxy => {:host => Billy.proxy.host, :port => Billy.proxy.port}
+        }
+        Capybara::Webkit::Driver.new(app, Capybara::Webkit::Configuration.to_hash.merge(options))
```

Comment:
It does allow you to config Capybara-Webkit. Yes, there could be a race-condition but I have not experienced any problems yet. I was thinking of open an issue about collisions between white/black lists and allowed URLs in Billy. But I want to be sure I know what's happening first. I'm using this branch and it works fine at the moment.